playwright/tests/pageobjects/abstract/Table.ts

```diff
@@ -15,13 +17,34 @@ export class Table {
     return new Row(this.rows, index);
   }
 
+  async focusTable() {
+    await this.table.focus();
+  }
+
   async expectToHaveValues(...values: Array<Array<string>>) {
     await expect(this.rows).toHaveCount(values.length);
     for (let i = 0; i < values.length; i++) {
       await this.row(i).expectToHaveValues(...values[i]);
     }
   }
 
+  async expectToBeSelected(indexes: Array<number>) {
+    for (let i = 0; i < indexes.length; i++) {
+      await this.row(indexes[i]).expectToBeSelected();
+    }
+  }
+
+  async expectRowToBeAtPosition(position: number, name: string) {
+    const cell = this.row(position).column(0).locator;
+    await expect(cell).toHaveText(name);
+  }

```

Comment:
I would not create a method for this and just call this chain directly:
```typescript
await expect(editor.table.row(0).column(0).locator).toHaveText("test");
```

This might even be enough already:
```typescript
await expect(editor.table.row(0).locator).toHaveText("test");
```

Or at least rename it to something like `expectRowToContain`. Makes more sense to me.